### PR TITLE
Update ValidatesAttributes.php

### PR DESCRIPTION
**Pull request by Norgul**
Bug fix for [this issue](https://github.com/laravel/framework/issues/36952)

### CODE REVIEW

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 1]
```diff
@@ -1563,6 +1563,19 @@ public function parseDependentRuleParameters($parameters)
         return [$values, $other];
     }
 
+    /**
+     * Check if parameter should be converted to boolean.
+     *
+     * @param $parameter
```
Comment (GrahamCampbell) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php:1569]: invalid syntax

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by GrahamCampbell** [comment]
Please provide a proper PR title.

**Comment by driesvints** [comment]
@Norgul please provide a proper title and pr description like the PR template says when you submit a PR.

**Comment by taylorotwell** [comment]
I am closing this pull request because it lacks sufficient **explanation**, **tests**, or both. It is difficult for us to merge pull requests without these things because the change may introduce breaking changes to the framework.

Feel free to re-submit your change with a **thorough** explanation of the feature and tests - integration tests are preferred over unit tests. Please include it's benefit to end users; the reasons it does not break any existing features; how it makes building web applications easier, etc.

Thanks!

**Pull request closed without merging**